testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusTest.java

```diff
@@ -96,8 +96,10 @@ public void testRuntime(TestInfo testInfo, List<CodeQuarkusExtensions> extension
             appendln(whatIDidReport, "Download URL: " + download(extensions, zipFile, 11));
             LOGGER.info("Unzipping...");
             unzipLog = unzip(zipFile, GEN_BASE_DIR);
-            LOGGER.info("Removing repositories and pluginRepositories from pom.xml ...");
-            removeRepositoriesAndPluginRepositories(appDir + File.separator + "pom.xml");
+            if (StringUtils.isBlank(System.getProperty("gh.actions"))) {
```

Comment:
I don't understand, what is going on here. Could you please give me a hint?

Also, in similar PR https://github.com/quarkus-qe/quarkus-startstop/pull/247 in `main`, the changes make even less sense to me: see [1], where the same statement appears both inside and outside of an `if`.

[1] https://github.com/quarkus-qe/quarkus-startstop/pull/247/files#diff-ae3062868d9d85b70aa0591f0df6858351562018539597a118013f09402bdfacR103

Reply:
The idea is to remove the definitions of additional maven repository when NOT running in GH CI where we explicitly define `gh.actions` property.
We want to remove the definition when we have product maven repo zip to ensure that all the needed artifacts are in that zip.
On GH CI we want to pull bits from https://maven.repository.redhat.com/ga ...

PR https://github.com/quarkus-qe/quarkus-startstop/pull/247 is incorrect, thanks for spotting.

Reply:
Thanks for the explanation, it makes total sense.

Does this change relate to the previous CI failures somehow?

Reply:
I don't think so, Daily CI runs on main, I's seen some GH limits fails when setting up GraalVM for Windows